Library/Core/UnoCore/Targets/iOS/@(Project.Name).xcodeproj/project.pbxproj

```diff
@@ -458,10 +458,31 @@
 #if @(LIBRARY:Defined)
                 MACH_O_TYPE = mh_dylib;
 #endif
+
+#if @(Project.iOS.SpecialCapabilities.CPPImport:IsSet)
```

Comment:
I'm not so fond of the `SpecialCapabilities`-namespace here. It seems a bit odd IMO; the rest of the settings under `Project.iOS` are just as "special", no?

Reply:
This namespace proposed by me, we can pick the better name ;)

Reply:
I think just `Project.iOS.SpecialCapabilities.CPPImport` is good enough...